flake.nix

```diff
@@ -165,39 +165,25 @@
         };
       };
 
-      tests-fpga = pkgs.writeShellApplication {
-        name = "tests-fpga";
-        runtimeInputs = [pythonEnv];
-        text = ''
-          set +u
-          if [ -z "$1" ]; then
-            echo "Please provide the tty device location (e.g. /dev/ttyUSB2)" \
-              "as the first argument."
-            exit 2
-          fi
-          ${./util/test_runner.py} -t 30 fpga "$1" \
-            --elf-file ${sonata-system-software}/bin/test_runner \
-            --tcl-file ${./util/sonata-openocd-cfg.tcl}
-          ${./util/test_runner.py} -t 600 fpga "$1" \
-            --uf2-file ${cheriot-rtos-test-suite}/share/test-suite.uf2
-        '';
+      bitstream = import nix/bitstream.nix {
```

Comment:
I think this is one commit to early.

Reply:
Good catch, git absorb sometimes mess think up.